sources/mysql/toddl_test.go

```diff
@@ -25,6 +25,103 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestToSpannerTypeInternal(t *testing.T) {
+
+	_, errCheck := toSpannerTypeInternal(schema.Type{"bool", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"bool", []int64{1, 2, 3}, []int64{1, 2, 3}}, `INT64`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"tinyint", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"tinyint", []int64{1, 2, 3}, []int64{1, 2, 3}}, `INT64`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"double", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"float", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"decimal", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"bigint", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"int", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck == nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"bit", []int64{1, 2, 3}, []int64{1, 2, 3}}, `STRING`)
+	if errCheck != nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"char", []int64{1, 2, 3}, []int64{1, 2, 3}}, `BYTES`)
+	if errCheck != nil {
+		t.Errorf("Error in timestamp string")
+	}
+	_, errCheck = toSpannerTypeInternal(schema.Type{"char", []int64{}, []int64{1, 2, 3}}, `BYTES`)
+	if errCheck != nil {
+		t.Errorf("Error in timestamp string")
```

Comment:
Is the error message correct?
Please check other files as well.

Reply:
done